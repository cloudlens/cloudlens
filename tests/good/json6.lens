stream when(e) {
    print(e.message)
}
