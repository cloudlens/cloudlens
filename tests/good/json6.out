6 pommes
2 apples
42
4 manzana
